### PR TITLE
Remove app

### PR DESCRIPTION
**Pull request by hackaugusto**
This removes the `App` class, it was just a proxy class without any useful logic.

### CODE REVIEW

File: raiden/app.py [hunk 1]
```diff
@@ -1,97 +0,0 @@
...
-        self.raiden = raiden
-
-    def __repr__(self) -> str:
-        return "<{} {}>".format(self.__class__.__name__, to_checksum_address(self.raiden.address))
-
-    def start(self) -> None:
-        """ Start the raiden app. """
-        if self.raiden.stop_event.is_set():
-            self.raiden.start()
-            log.info("Raiden started", node=to_checksum_address(self.raiden.address))
```
Comment (palango) [inline raiden/app.py:91]: Should we keep those log lines?

File: raiden/waiting.py [hunk 2]
```diff
@@ -188,37 +187,37 @@ def wait_for_participant_deposit(
...
+        target_address=app_deposit.address,
         target_balance=total_deposit,
         retry_timeout=retry_timeout,
     )
 
 
 def wait_both_channel_deposit(
-    app_deposit: "App",
-    app_partner: "App",
+    app_deposit: "RaidenService",
```
[the remark this answers is not in the document]
Comment (hackaugusto) [reply raiden/waiting.py:219]: what should I use as a new name?

[the remark this answers is not in the document]
Comment (palango) [reply raiden/waiting.py:219]: I thought about removing the `app` there, but we use it in basically every integration test. So just leave it.

File: raiden/app.py (hunk 1 above)
Comment (hackaugusto) [reply raiden/app.py:91]: There are equivalent log lines for the service: https://github.com/raiden-network/raiden/blob/develop/raiden/raiden_service.py#L447

**Pull request merged**